src/dbg/simplescript.cpp

```diff
@@ -692,6 +694,12 @@ bool scriptcmdexec(const char* command)
     case STATUS_PAUSE:
     case STATUS_CONTINUE:
         break;
+    case STATUS_CONTINUE_BRANCH:
+        if(!bIsRunning)
+        {
+            scriptrun(scriptIp);

```

Comment:
I guess we need to pass down that `scriptcmd` triggered this `scriptrun` and not trigger the popup in that case?